test/steward_multi_token_deposit.js

```diff
@@ -13,6 +13,17 @@ const waitTillBeginningOfSecond = () => new Promise(resolve => {
 })
 
 // todo: test over/underflows
+const NUM_SECONDS_IN_YEAR = '31536000'
+const PATRONAGE_DENOMINATOR = '1'
+
+
+const multiPatronageCalculator = (timeInSeconds, tokenArray) => {
```

Comment:
```suggestion
const multiPatronageCalculator = (timeInSeconds, tokenArray) => {
  const totalPatronage = tokenArray.reduce(
    (totalPatronage, token) =>
      totalPatronage.add(
        (new BN(token.price))
          .mul(new BN(timeInSeconds))
          .mul(new BN(token.patronageNumerator))
          .div(new BN(PATRONAGE_DENOMINATOR))
          .div(new BN(NUM_SECONDS_IN_YEAR))
      )
    , new BN('0'));
  return totalPatronage
}
```
This is much easier to read.